app/assets/stylesheets/style.css

```diff
@@ -527,7 +527,14 @@ textarea {
     margin-top:45px !important;
     text-align:center;
 }
-
+.btn-danger {
```

Comment:
This is already defined in bootstrap-sass in the `/assets/stylesheets/bootstrap/_buttons.scss` file that defines the style:
```
// Danger and error appear as red
.btn-danger {
  @include button-variant($btn-danger-color, $btn-danger-bg, $btn-danger-border);
}
```

and the `assets/stylesheets/bootstrap/_variables.scss` file that defines the actual colors:
```
$brand-danger:          #d9534f !default;

...

$btn-danger-color:               #fff !default;
$btn-danger-bg:                  $brand-danger !default;
$btn-danger-border:              darken($btn-danger-bg, 5%) !default;
```

We should just use those styles.

Reply:
I know this, but this is overwritten by wpcf7 for this particulat button, thats why I add it again with the !important addition.

Reply:
aha.  (I figured that you must know that, but perhaps had just forgotten momentarily or something.)